src/platform/platform.dom.js

```diff
@@ -119,7 +119,7 @@ const supportsEventListenerOptions = (function() {
 				supports = true;
 			}
 		});
-		window.addEventListener('e', null, options);
+		window.addEventListener('click', function() {}, options);
```

Comment:
Looks like this was changed a bit in MDN:
```js
let passiveSupported = false;

try {
  const options = {
    get passive() { // This function will be called when the browser attempts to access the passive property.
      passiveSupported = true;
      return false;
    }
  };

  window.addEventListener("test", null, options);
  window.removeEventListener("test", null, options);
} catch(err) {
  passiveSupported = false;
}
```
There is also a note: 
> (Note that handleEvent() is ignored on event listeners that aren't called.)

Reply:
There are two issues with that sample that won't work. The first is that `"test"` is not a valid event type. The second is that `null` is not a valid second argument. That's what I was changing here

I'm not quite sure what you're suggesting be done differently. Are you saying I also need to add the following?
```
window.removeEventListener('click', function() {}, options);
```

Reply:
If using a valid event, there is the possibility it gets called also. So need the empty function for that. Does not make the core any better or safer though, only a bit bigger. Another // @ts-ignore.
I'd use the MDN version though, where the event listener is also removed. And ES6 syntax for the getter is used. And a return value, so can omit that // eslint directive.

Reply:
done